presto-clickhouse/src/main/java/com/facebook/presto/plugin/clickhouse/optimization/function/OperatorTranslators.java

```diff
@@ -33,8 +35,8 @@ private OperatorTranslators()
     }
 
     @ScalarOperator(ADD)
-    @SqlType(StandardTypes.BIGINT)
```

Comment:
I think the file we want to change is in [presto-base-jdbc](https://github.com/prestodb/presto/blob/master/presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/optimization/function/OperatorTranslators.java#L29)

Reply:
Ok, I can update all the cases in the JDBC file, should we ultimately change all operator translator classes?

Reply:
Yes, we should support all translators eventually. I am not familiar with the SQL dialect that Clickhouse supports, but I think the basic operators + types should translate and work with it. Feel free to take it up in another PR if this reduces the testing overhead

Reply:
Ok, will do once this one gets merged!